godotopenxrkhr/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="org.godotengine.openxrloaders.khr"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <uses-permission android:name="org.khronos.openxr.permission.OPENXR_SYSTEM" />
+
+    <application>
+        <meta-data
+            android:name="org.godotengine.plugin.v1.GodotOpenXRKHR"
+            android:value="org.godotengine.openxrloaders.khr.GodotOpenXRKHR" />
+
+        <activity android:name="com.godot.game.GodotApp" android:exported="true" tools:ignore="MissingClass">
+            <intent-filter>
```

Comment:
Tried adding tools:node="merge" to intent-filter tag but bundletool refuses to merge the toplevel intent-filter with this. Hence the work-around below.